indico/modules/receipts/client/js/printing/TemplateParameterEditor.jsx

```diff
@@ -68,6 +84,19 @@ function CustomField({name, value, type, attributes, validations, onChange}) {
         required={validations.required}
       />
     );
+  } else if (type === 'image') {
+    return (
+      <EventImageSelectField
+        label={attributes.label}
+        name={name}
+        value={value}
+        required={validations.required}
+        loading={loadingImages}
+        onChange={onChange}
+        eventImages={eventImages}
+        onRefresh={fetchImages && (() => fetchImages())}
```

Comment:
it would work, but the `onRefresh` is also passing the JS event from the icon's onClick, so this is an explicit way to "ditch" those unnecessary props